articleModel.js

```diff
@@ -29,44 +28,32 @@ exports.selectArticleById=(id)=>{
 }
 
 exports.selectAllArticles=()=>{
-  const  queryString=`SELECT article_id,title,topic,A.author,
-  created_at,votes,
-   U.avatar_url AS article_img_url
+  const  queryString=`SELECT A.article_id,A.title,A.topic,
```

Comment:
Chris, looks really good. I would question if we need the dot notation referring to articles as 'A'?